app/controllers/facility_statements_controller.rb

```diff
@@ -72,6 +72,16 @@ def create
     redirect_to action: :new
   end
 
+  # GET /facilities/:facility_id/statements/:id/resend_emails
```

Comment:
I don't think that this should be a "GET" action, because it has side-effects (sending an e-mail)